examples/java-matter-controller/java/src/com/matter/controller/commands/pairing/PairOnNetworkLongImWriteCommand.java

```diff
@@ -0,0 +1,79 @@
+package com.matter.controller.commands.pairing;
+
+import chip.devicecontroller.ChipDeviceController;
+import chip.devicecontroller.GetConnectedDeviceCallbackJni.GetConnectedDeviceCallback;
+import chip.devicecontroller.WriteAttributesCallback;
+import chip.devicecontroller.model.AttributeWriteRequest;
+import chip.devicecontroller.model.ChipAttributePath;
+import com.matter.controller.commands.common.CredentialsIssuer;
+import java.util.ArrayList;
+import java.util.logging.Level;
+
+public final class PairOnNetworkLongImWriteCommand extends PairingCommand
+    implements WriteAttributesCallback, GetConnectedDeviceCallback {
+  private static final int MATTER_PORT = 5540;
+  private long devicePointer;
+
+  public PairOnNetworkLongImWriteCommand(
+      ChipDeviceController controller, CredentialsIssuer credsIssue) {
+    super(
+        controller,
+        "onnetwork-long-im-write",
+        PairingModeType.ON_NETWORK,
+        PairingNetworkType.NONE,
+        credsIssue,
+        DiscoveryFilterType.LONG_DISCRIMINATOR);
+  }
+
+  @Override
+  protected void runCommand() {
+    // boolean true for tlv
+    byte[] booleanTLV = {0x09};
```

Comment:
Since https://github.com/project-chip/connectedhomeip/pull/25411 was merged, would it make sense here to switch to `TlvWriter` to build the payload?

Reply:
We would update this when converting this file to kt. thanks

Reply:
+import chip.tlv.AnonymousTag;
+import chip.tlv.TlvWriter;
 
 public final class PairOnNetworkLongImWriteCommand extends PairingCommand implements GetConnectedDeviceCallback{
   private static final int MATTER_PORT = 5540;
@@ -53,10 +55,12 @@ public final class PairOnNetworkLongImWriteCommand extends PairingCommand implem
   @Override
   protected void runCommand() {
     // boolean true for tlv
-    byte[] booleanTLV = {0x09};
+    //TlvWriter booleanTLV;
+    AnonymousTag tag;
+    booleanTLV.put(tag, true);
     AttributeWriteRequest attribute = AttributeWriteRequest.newInstance(    /* endpointId= */ 0,
     CLUSTER_ID_BASIC,
-    ATTR_ID_LOCAL_CONFIG_DISABLED, booleanTLV);
+    ATTR_ID_LOCAL_CONFIG_DISABLED, booleanTLV.getEncoded());
     ArrayList<AttributeWriteRequest> attributeList = new ArrayList<>();
     attributeList.add(attribute);

FYI, this is what i did, call this kotlin method from java